example/backend.go

```diff
@@ -24,10 +24,11 @@ func main() {
 
 	// Serves static pages
 	log.Println("Serving /static - serves html files for testing")
-	mux.Get("/static/:param", func(w http.ResponseWriter, r *http.Request) {
-		log.Printf("Filepath:%s\n",r.URL.Path[1:])
+	mux.Get("/:param", func(w http.ResponseWriter, r *http.Request) {
+		path := "static/" + r.URL.Path[1:]
```

Comment:
This fix was about a workaround involving the /static path of the example backend server. This was the first code I worked on in the project, I had no concept of the fact that the static path should be root in this server to test the backend. Fixing that was easy.